src/app.ts

```diff
@@ -115,8 +115,8 @@ export function createHandle (stack: Stack, options: AppOptions): PHandle {
       if (type === 'string') {
         return send(res, val, MIMES.html)
       } else if (type === 'object' || type === 'boolean' || type === 'number' /* IS_JSON */) {
-        if (val && val.buffer) {
-          return send(res, val)
+        if (val && (val.buffer || isStream(val))) {
+          return send(res, val, MIMES.html)
```

Comment:
Html mime is probably not a safe assumption. Stream responses could be binary also for sending assets

Reply:
Perf: We already know data is stream when calling send. isStream check could be removed for send function

Reply:
- For the html mime, we can explicitly set to `application/octet-stream` by default (which is better than undefined), what do you think? 
- For the `isStream` call, if we don't, it will run in the else branch and the content will be stringified